### DISCUSSION TITLE
Propose for a componet like 'PickList'

### DISCUSSION THREAD

**Discussion by donald001** [quasarframework/quasar#16859] [Ideas / Proposals] [answered]
Most of our components are single sided. Sometimes, it would be better to have an option for us to choose items, or move one item from one list to another list. I wish we could have such a component like it:  [pickList](https://primevue.org/picklist/)
![image](https://github.com/quasarframework/quasar/assets/38281146/5cdee657-5018-4a67-8483-965232e4880e)

**Comment by joakweb** [comment]
+1 to this. Quasar is the main front lib we use in my company and I just looked up for something like this and found it on PrimeVue instead.

**Comment by metalsadman** [comment, marked as answer]
I've created something like that based off Luke's codepen. It's a q-card container with q-scroll-area and q-list as content.
![picker](https://github.com/quasarframework/quasar/assets/31038838/a6ed18f6-b0e2-4f95-904a-f6587ab6cc03)

https://codepen.io/metalsadman/pen/GRyRjZB
